code/datums/traits/negative.dm

```diff
@@ -479,24 +479,28 @@
 	desc = "Talking to people is very difficult for you, and you often stutter or even lock up."
 	value = -1
 	gain_text = "<span class='danger'>You start worrying about what you're saying.</span>"
-	lose_text = "<span class='notice'>You feel easier about talking again.</span>" //if only it were that easy!
+	lose_text = "<span class='notice'>You feel comfortable with talking again.</span>" //if only it were that easy! (second this - etherware)
```

Comment:
Comments should be informational rather than degrading to yourself
```suggestion
	lose_text = "<span class='notice'>You feel comfortable with talking again.</span>" //if only it were that easy!
```